boxnav/boxnav/boxnavigator.py

```diff
@@ -398,13 +387,14 @@ def execute_navigator_action(self) -> tuple[Action, Action]:
             self.images_saved += 1
             self.latest_image_filepath = f"{self.image_directory}/{self.trial_num:03}_{self.images_saved:06}_{angle}.{self.image_extension}"
 
-            self.ue.save_image(self.latest_image_filepath, delay=0.25)
+            self.ue.save_image(self.latest_image_filepath, delay=self.image_delay)
+            self.image_delay = 0.25
```

Comment:
```suggestion
            # Lower the delay after the first image since UE is warmed up
            self.ue.save_image(self.latest_image_filepath, delay=self.image_delay)
            self.image_delay = 0.25
```